src/DynamoApplications/StartupUtils.cs

```diff
@@ -207,33 +207,34 @@ internal static Version GetVersionFromASMPath(string asmPath, string searchPatte
         /// <returns></returns>
         public static DynamoModel MakeModel(bool CLImode, string asmPath)
         {
-            if (!Directory.Exists(asmPath)){
-                throw new FileNotFoundException($"{nameof(asmPath)}:{asmPath}");
```

Comment:
Was this exception being caught somewhere before you enclosed it in a `try-catch`?

Reply:
no, the application would crash.